nbbuild/travis/scripting.sh

```diff
@@ -41,24 +41,24 @@ $GRAALVM/bin/gu install ruby
 
 JAVA_HOME=$GRAALVM
 echo "ant -f platform/api.scripting/build.xml test"
-hide-logs.sh ant -f platform/api.scripting/build.xml test
+ant -f platform/api.scripting/build.xml test
 JAVA_HOME=$GRAALVM
 echo "ant -f ide/libs.graalsdk/build.xml test"
-hide-logs.sh ant -f ide/libs.graalsdk/build.xml test
+ant -f ide/libs.graalsdk/build.xml test
 JAVA_HOME=$GRAALVM
 echo "ant -f platform/core.network/build.xml test"
-hide-logs.sh ant -f platform/core.network/build.xml test
+ant -f platform/core.network/build.xml test
 JAVA_HOME=$GRAALVM
 echo "ant -f webcommon/libs.graaljs/build.xml test"
-hide-logs.sh ant -f webcommon/libs.graaljs/build.xml test
+ant -f webcommon/libs.graaljs/build.xml test
 JAVA_HOME=$GRAALVM
 echo "ant -f profiler/profiler.oql/build.xml test"
-hide-logs.sh ant -f profiler/profiler.oql/build.xml test
+ant -f profiler/profiler.oql/build.xml test
 JAVA_HOME=$GRAALVM
 echo "ant -f java/nashorn.execution/build.xml test"
-hide-logs.sh ant -f java/nashorn.execution/build.xml test
+ant -f java/nashorn.execution/build.xml test
 JAVA_HOME=$GRAALVM
 echo "ant -f java/debugger.jpda.truffle/build.xml test"
-hide-logs.sh ant -f java/debugger.jpda.truffle/build.xml test
+ant -f java/debugger.jpda.truffle/build.xml test
```

Comment:
i would like to take this whole script and convert this into gh actions

this would map directly to steps. Makes it easier to see what fails (log split per step), allows to add retry per test etc. Disadvantage is that it makes it less portable.